dub.json

```diff
@@ -87,16 +87,7 @@
             "targetName": "agora-client",
             "mainSourceFile": "source/agora/cli/client/main.d",
             "excludedSourceFiles": [
-                "source/agora/api/Validator.d",
-                "source/agora/common/Config.d",
-                "source/agora/common/SCPHash.d",
-                "source/agora/network/*",
-                "source/agora/node/*",
-                "source/agora/utils/PrettyPrinter.d",
-                "source/agora/utils/SCPPrettyPrinter.d",
-                "source/scpp/*",
-                "source/scpd/*",
-                "source/agora/consensus/protocol/*",
```

Comment:
That doesn't look right.

Reply:
Re-add only that part to the ExcludedSourceFiles